app/components/Table.tsx

```diff
@@ -58,7 +59,7 @@ export default function Table<T extends Record<string, unknown>>({
           <tr key={i}>
             {columns.map((col) => (
               // todo: is no-wrap / side-scrolling ok?
-              <td key={String(col.path)} style={{ whiteSpace: "nowrap" }}>
+              <td key={String(col.path)} style={col.style}>
```

Comment:
this enables our FRB recommendation text to wrap and makes use of this cool style prop that Mateo added in https://github.com/cityofaustin/vision-zero/pull/1659